packages/oscal-react-library/src/components/OSCALMetadataCommon.tsx

```diff
@@ -45,6 +45,37 @@ const OSCALMetadataSectionInfoHeader = styled(Typography)`
   align-items: center;
 ` as typeof Typography;
 
+const TELEPHONE_ICON_MAPPING = {
+  home: <HomeIcon />,
+  mobile: <SmartphoneIcon />,
+  office: <BusinessIcon />,
+};
+
+const ADDRESS_ICON_MAPPING = {
+  home: <HomeIcon />,
+  work: <BusinessIcon />,
+};
+
+const UNKNOWN_TYPE_ICON = <HelpOutlineIcon />;
+
+function getIconOfType(
+  mapping: Record<string, ReactNode> | undefined,
```

Comment:
I think you can be more specific and use `SvgIcon`?

https://github.com/mui/material-ui/blob/master/packages/mui-material/src/utils/createSvgIcon.d.ts

Reply:
Maybe with a `typeof` in here somewhere?